src/commands/teardown.rs

```diff
@@ -24,7 +24,20 @@ impl Teardown {
         debug!("{:?}", "Tearing down..");
         let _network_options = match network::types::NetworkOptions::load(&input_file) {
             Ok(opts) => opts,
-            Err(e) => panic!("{}", e),
+            Err(e) => {
```

Comment:
this could be simplified into 
```rust
        let network_options = match network::types::NetworkOptions::load(&input_file) {
            Ok(opts) => opts,
            Err(e) => {
                return Err(Box::new(network::error::NetavarkError{ error: format!("{}", e), errno: 1}));
            }
        };
```